test/calculate-totals.test.js

```diff
@@ -0,0 +1,302 @@
+const expect = require("chai").expect
+const proxyquire = require("proxyquire")
+const reportFixture = require("./fixtures/report")
+const dataFixture = require("./fixtures/data")
+const calculateTotals = require("../src/calculate-totals")
+
+proxyquire.noCallThru()
```

Comment:
By default, if a stub does not implement a function, the stub will pass a function call through the stub to the function in the underlying module. This can be helpful, but I've found that it's not worth it if you don't need it b/c it leads to tough to find problems when things are renamed or misspelled. Calling `noCallThru()` on `proxyquire` disables that functionality.